node/pkg/allocateip/allocateip.go

```diff
@@ -498,6 +498,7 @@ func updateNodeWithAddress(ctx context.Context, c client.Interface, nodename str
 func removeHostTunnelAddr(ctx context.Context, c client.Interface, nodename string, attrType string) {
 	var updateError error
 	logCtx := getLogger(attrType)
+	logCtx.WithField("Node", nodename).Debug("Remove tunnel addresses")
```

Comment:
I think elsewhere we use lowercase field names, so this guy should be "node" for consistency.

Reply:
The one place I looked (line 251, in `ensureHostTunnelAddress()` was uppercase, heh. I'll change both to lowercase, thanks for catching this!

Reply:
whoops, of course we're doing this elsewhere :facepalm: 

I think lowercase is still _more_ consistent, though.